lib/db.js

```diff
@@ -529,7 +588,11 @@ module.exports = (
         uid: sessionToken && sessionToken.uid
       }
     )
-    return this.pool.del('/sessionToken/' + sessionToken.id)
+    const promises = [this.pool.del('/sessionToken/' + sessionToken.id)]
+    if (this.redis) {
+      promises.push(this.redis.del(sessionToken.uid))
```

Comment:
Will this delete all of the session tokens for that `uid`? I think we only want to delete an individual session token at this point.